DatadogSessionReplay/Sources/Processor/SRDataModelsBuilder/WireframesBuilder.swift

```diff
@@ -17,7 +17,7 @@ internal typealias WireframeID = NodeID
 /// It is used by the player to reconstruct individual elements of the recorded app UI.
 ///
 /// Note: `WireframesBuilder` is used by `Processor` on a single background thread.
-internal class WireframesBuilder {
+@_spi(Internal) public class SessionReplayWireframesBuilder {
```

Comment:
`@_spi` comes with a risk, but it looks like some [other SDKs](https://github.com/search?q=repo%3Astripe%2Fstripe-ios+spi&type=code) are using it.

[Nice read from PSPDFKit folks](https://pspdfkit.com/blog/2023/swifts-approach-to-spi/) convinced me we could pursue that path.